test/specs/post.js

```diff
@@ -23,14 +23,14 @@ var defaultRelease = {
   repo: 'up',
   name: 'v1.0.0',
   tag_name: 'v1.0.0',
-  target_commitish: 'bar',
+  target_commitish: 'master',
   body: 'the log'
 }
 
 test('full post run', function (t) {
   t.test('in debug mode w/o token', function (tt) {
     post({
-      options: {debug: true},
+      options: {debug: true, branch: 'master'},
```

Comment:
can you explain why this change was needed? Just want to make sure that merging this change won’t break something else, also want to better understand you changes :)